test/spec/MfaVerify_spec.js

```diff
@@ -1808,6 +1815,59 @@ function (Q,
           });
         });
       });
+
+      Expect.describe('Security Key (U2F)', function () {
+        itp('shows the right beacon for Security Key (U2F)', function () {
+          return setupU2F().then(function (test) {
+            expectHasRightBeaconImage(test, 'mfa-u2f');
+          });
+        });
+
+        itp('shows the right title', function () {
+          return setupU2F().then(function (test) {
+            expectTitleToBe(test, 'Security Key (U2F)');
+          });
+        });
+
+        itp('shows a spinner while waiting for u2f challenge', function () {
```

Comment:
Do we have a test that verifies the spinner goes away when we get the response? Same goes for the enroll flow.